docs/blog/posts/2024-04-20-expanding-trusted-publisher-support.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Expanding Trusted Publisher Support
+description: Announcing additional Trusted Publishing providers
+authors:
+  - di
+date: 2024-04-20
```

Comment:
Yeah it will publish in the future. This is merging into https://github.com/pypi/warehouse/pull/15192, we'll update the final date there once we've coordinated with ActiveState and GitLab.